iree/hal/cts/cts_test_base.h

```diff
@@ -30,14 +31,21 @@ namespace cts {
 
 // Common setup for tests parameterized across all registered drivers.
 class CtsTestBase : public ::testing::TestWithParam<std::string> {
+ public:
+  static std::vector<std::string> EnumerateAvailableDrivers() {
+    static std::once_flag register_once;
+    std::call_once(register_once, [] {
```

Comment:
doesn't have to be done here (unless you want to!), but we could move this call-once into the register_all_drivers fn to make it less sharp to users - C has call_once too so we can use it in there https://en.cppreference.com/w/c/thread/call_once

Reply:
Would be useful, future cleanup.

Reply:
added to #3843